src/main.h

```diff
@@ -96,7 +97,7 @@ static const unsigned int UNDOFILE_CHUNK_SIZE = 0x100000; // 1 MiB
 /** Maximum number of script-checking threads allowed */
 static const int MAX_SCRIPTCHECK_THREADS = 16;
 /** -par default (number of script-checking threads, 0 = auto) */
-static const int DEFAULT_SCRIPTCHECK_THREADS = 0;
+static const int DEFAULT_SCRIPTCHECK_THREADS = 2;
```

Comment:
Why did we limit the threads here to 2 instead of auto?

Reply:
It's default value, not max. Improves performance